stargaze-1/v2_0_0_UPGRADE.md

```diff
@@ -0,0 +1,27 @@
+# v2.0.0 Upgrade
+
+The upgrade is scheduled for block `1317621`, which should be about _15:30PM UTC on 25th January 2022_.
+
+These instructions assume you are running Cosmovisor. Most mainnet validators are running Cosmovisor, and [a setup guide can be found here](https://docs.junonetwork.io/validators/setting-up-cosmovisor).
+
```

Comment:
Can we add a warning saying that cosmovisor is configured to take a full backup unless `UNSAFE_SKIP_BACKUP=true`

Reply:
Yes.

Reply:
Added.